release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -2813,6 +2813,52 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* The `oc-mirror v2` was not using the correct filtered catalog during its operations leading to errors such as including more operators than specified in the configuration, and trying to connect to the catalog registry during disk-to-mirror workflows even in air-gapped environments. With this release, the correct filtered catalog is used. (link:https://issues.redhat.com/browse/OCPBUGS-57784[OCPBUGS-57784])
+
+* Previously, the OpenShift Lightspeed UI would disappear when the *Create Project* modal was opened or when modals on the Networking pages were triggered. This was due to the modals using the `useModal` hook causing the modals to overwrite each other. With this release, the modals no longer overwrite each other allowing multiple UI elements to be displayed simultaneously. (link:https://issues.redhat.com/browse/OCPBUGS-57755[OCPBUGS-57755])
+
+* Previously, the HAProxy configuration used the `/version` endpoint for health checks causing unreliable health checks to be generated. With this release, the liveness probe is customized to use `/livez?exclude=etcd&exclude=log` on IBM Cloud for more accurate health checks avoiding disruptions due to inappropriate probe configurations on Hypershift, while retaining `/version` for other platforms. (link:https://issues.redhat.com/browse/OCPBUGS-57485[OCPBUGS-57485])
+
+* Previously, the installer failed when Amazon Web Service (AWS) credentials were not found and the survey was attempting AWS regions preventing users from creating the install-config. With this release, the installer no longer fails when AWS credentials are not set, allow users to input them during the survey. (link:https://issues.redhat.com/browse/OCPBUGS-57394[OCPBUGS-57394])
+
+* Previously, cloning the Red Hat OpenShift Console Persistent Volume Claim (PVC) resulted in an error due to an unsupported unit *B* for storage size. Because of this, users encountered errors when cloning the Red Hat OpenShift console PVC due to incorrect parsing of the storage size unit. With this release, support for *B* as unit of storage size has been removed from the Red Hat OpenShift console PVC. (link:https://issues.redhat.com/browse/OCPBUGS-57391[OCPBUGS-57391])
+
+* Previously, `OLMv1` was used to install Operators with the `olm.maxOpenShiftVersion` set to  `4.19`.  Due to an issue with the `OLMv1` parsing logic for floating-point formatted `olm.maxOpenShiftVersion`values, the system failed to prevent upgrades to Red Hat OpenShift 4.20. With this release, the parsing logic for  `olm.maxOpenShiftVersion` has been corrected preventing upgrades to Red Hat OpenShift 4.20 when Operators with `olm.maxOpenShiftVersion:4.19` are installed. (link:https://issues.redhat.com/browse/OCPBUGS-56852[OCPBUGS-56852])
```

Comment:
```suggestion
* Previously, `OLMv1` was used to install Operators with the `olm.maxOpenShiftVersion` set to  `4.19`.  Due to an issue with the `OLMv1` parsing logic for floating-point formatted `olm.maxOpenShiftVersion` values, the system failed to prevent upgrades to {product-title} 4.20. With this release, the parsing logic for  `olm.maxOpenShiftVersion` has been corrected, preventing upgrades to {product-title} 4.20 when Operators with `olm.maxOpenShiftVersion:4.19` are installed. (link:https://issues.redhat.com/browse/OCPBUGS-56852[OCPBUGS-56852])
```